background/background.js

```diff
@@ -295,99 +297,34 @@ window.addEventListener('storageReady', function _() {
     }));
 });
 
-// *************************************************************************
-{
-  const getStylesForFrame = (msg, sender) => {
-    const stylesTask = getStyles(msg);
-    if (!sender || !sender.frameId) return stylesTask;
-    return Promise.all([
-      stylesTask,
-      getTab(sender.tab.id),
-    ]).then(([styles, tab]) => {
-      if (tab) styles.exposeIframes = tab.url.replace(/(\/\/[^/]*).*/, '$1');
-      return styles;
-    });
-  };
-  const updateAPI = (_, enabled) => {
-    window.API_METHODS.getStylesForFrame = enabled ? getStylesForFrame : getStyles;
-  };
-  prefs.subscribe(['exposeIframes'], updateAPI);
```

Comment:
This should be fixed in #510.

Reply:
It needs a new API like `API.getTabDomain` to fetch the domain of the parent window. I'll fix this after #510 is merged.